### DISCUSSION TITLE
Which File to put in?

### DISCUSSION THREAD

**Discussion by codegerbillin** [bnpr/Malt#364] [Q&A]
I downloaded my BlenderMalt File and instead of a zip file its a folder, ive tried installing it with the MaltRenderEngine.py but it doesn't do anything as far as I can see, nor does the shader show up. 

I am using 2.93.4 but it also doesn't work in 3.2.1

**Comment by Kolupsy** [comment]
If you are not building Malt yourself you should be downloading a release and install it like any other regular addon. The development version is only supported for the latest major blender release but you'll also get that information when you try to install the malt addon in an unsupported version. 

Also be sure to switch your render engine to Malt to get started

**Reply by codegerbillin** [reply]
Hmm im a bit confused, should Malt already be in my render engine before downloading a release?

**Reply by Kolupsy** [reply]
No. You install Malt like any other addon. But you use one of the releases:

Development: https://github.com/bnpr/Malt/releases/tag/Development-latest
Release: https://github.com/bnpr/Malt/releases/tag/Release-latest

This should download a .zip. Dont unpack it, just install it like you would install other addons: Edit>Preferences>Add-ons>Install

Make sure you clean up whatever did before that and relaunch blender before attempting to properly install Malt.

**Reply by codegerbillin** [reply]
Oh I understand that, but when I download it it unpacks the zip for me, so im not sure how to have/keep it as a zip.

**Reply by pragma37** [reply]
That's an issue with your browser or whatever you are using to download the file.
If there's something unzipping the file for you, just zip the BlenderMalt folder again.
